使用`python38`的环境，如果同时开了代理的话，`pip install`会报这个错误：

`after connection broken by 'ProxyError('Cannot connect to proxy.', FileNotFoundError(2, 'No such file or directory'))': /pypi/simple/pip/`



解决方法：

1. 临时关闭代理，打包完成后再重新打开；
2. 如果代理不能关，可以修改`%APPDATA%\pip\pip.ini`配置文件，添加一行`proxy=http://127.0.0.1:7890`到`global`配置下；